src/components/DonationTable/DonationTable.tsx

```diff
@@ -48,7 +48,7 @@ export interface DonationTableProps {
   emptyPlaceholder: React.ReactElement;
 }
 
-const StyledGrid = styled(DataGrid)(({ theme }) => ({
+export const StyledGrid = styled(DataGrid)(({ theme }) => ({
```

Comment:
Is there a reason to export these things? I don't see a place where they're being used.

Reply:
they are used here: src/components/Tool/Appeal/Modals/UpdateDonationsModal/DonationTable/DonationTable.tsx

Reply:
As I re-created the `donationsTable`, I wanted to keep as much of it the same as I could.